src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsVUAP.java

```diff
@@ -0,0 +1,76 @@
+package com.wikia.webdriver.testcases.adstests;
+
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.common.core.url.Page;
+import com.wikia.webdriver.common.templates.TemplateNoFirstLoad;
+import com.wikia.webdriver.common.contentpatterns.AdsContent;
+import com.wikia.webdriver.pageobjectsfactory.componentobject.ad.VUAP;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase.AdsBaseObject;
+import org.openqa.selenium.Dimension;
+import org.testng.annotations.Test;
+
+public class TestAdsVUAP extends TemplateNoFirstLoad {
+
+  private static final Dimension DESKTOP_SIZE = new Dimension(1920, 1080);
+  private static final String FANDOM_URL = "http://www.wikia.com/fandom";
+  private static final String PROJECT43 = "project43";
+  private static final String VUAP_ARTICLE = "SyntheticTests/VUAP";
+
+  private final String clickThroughUrl() {
+    return driver.getCurrentUrl();
+  }
+
+  @Test(
+          groups = "AdsVuapOasis"
```

Comment:
please add unique group to each tests so it's possible to run only one of them from Jenkins